main.go

```diff
@@ -107,11 +114,15 @@ func main() {
 		go statusService(statusAddr)
 	}
 
-	uri, err := url.Parse(fmt.Sprintf("relay://%s/?id=%s", extAddress, id))
+	uri, err := url.Parse(fmt.Sprintf("relay://%s/?id=%s&pingInterval=%s&networkTimeout=%s&sessionLimitBps=%d&globalLimitBps=%d&statusAddr=%s", extAddress, id, pingInterval, networkTimeout, sessionLimitBps, globalLimitBps, statusAddr))
```

Comment:
It's not though, it's just a bunch of values posted, with even less type safety than JSON. :) And the relay server will respond with the values in JSON later. But sure, not a huge deal either way.